pymodbus/server/async_io.py

```diff
@@ -269,6 +269,7 @@ def __init__(
         broadcast_enable=False,
         response_manipulator=None,
         request_tracer=None,
+        **kwargs,  # pylint: disable=unused-argument
```

Comment:
This is wrong, it just hides the problem, it allows you to think that loop is an acceptable parameter.

Please instead remove the loop parameter in the call.

Reply:
I found that in async_io.py ModbusSerialServer takes **kwargs while other types(Tcp, Tls,...) does not take this argument, so I think simply remove "loop" in the call may cause problems. Should I check server class before removing it?

Reply:
No the server classes get the loop internally with asyncio.get_current_loop(), so it no longer have an effect to add loop=

Kmarts still exist in some server classes due  to other parameters.

Reply:
Changes commited.